horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/RunServiceImpl.java

```diff
@@ -146,7 +147,7 @@ void onTestDeleted(TestDAO test) {
       log.debugf("Trashing runs for test %s (%d)", test.name, test.id);
       ScrollableResults results = Util.scroll(em.createNativeQuery("SELECT id FROM run WHERE testid = ?1").setParameter(1, test.id));
       while (results.next()) {
-         int id = (int) results.get(0);
+         int id = (int) results.get();
```

Comment:
`results.get()` should return an array, is the call returning a scalar value here that we can cast as an `int` to the id>